mkdocs.yml

```diff
@@ -19,6 +19,14 @@ nav:
   - udf_generation.md
   - contributing.md
   - resources.md
+  - Examples:
+      - "notebooks/create_udf.ipynb"
+      - "notebooks/example.ipynb"
```

Comment:
Shall we move this one to the first so that users can have a quick overview?

Reply:
Done